src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
+
+  const beforeForEach = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item !== '');
```

Comment:
The chained methods should have the same indentation to improve code readability. Align the '.map' and '.filter' methods with the '.split' method.

Reply:
if I do this, I will have more than 80 characters in a line. Maximum allowed is 80.